frontend/app-development/config/routes.tsx

```diff
@@ -46,7 +48,7 @@ export const routes: IRoute[] = [
     activeSubHeaderSelection: TopBarMenu.About,
     activeLeftMenuSelection: 'Om appen',
     menu: 'about',
-    subapp: Administration,
+    subapp: shouldDisplayFeature('newAdministration') ? Administration : LegacyAdministration,
```

Comment:
Love that you moved the feature flag here! 👏 I thinks it makes clean code by separating the new and legacy administration page like this. 👏